src/settings/DomainDnsStatus.js

```diff
@@ -30,6 +30,16 @@ export class DomainDnsStatus {
 		return this.status.getLoaded()
 	}
 
+	areRecordsFine(): boolean {
+		if (!this.status.isLoaded()
+			|| this.status.getLoaded().checkResult !== CustomDomainCheckResult.CUSTOM_DOMAIN_CHECK_RESULT_OK) {
```

Comment:
Isn't this called from the promise retunred from `loadCurrentStatus()` so it should always be loaded?

Reply:
> Isn't this called from the promise retunred from `loadCurrentStatus()` so it should always be loaded?
Yes, currently it is save without this check, but I added it because the check is cheap and will avoid problems in case the method is used elsewhere in the future.